### ISSUE TITLE
Improve marketing site accesibility

### ISSUE THREAD

**Issue by kurealnum** [toddagriscience/nightcrawler#499]
### Problem Statement

Our accessibility isn't too great. Each page of the marketing site should be checked with https://www.accessibilitychecker.org/. The fixes on each page should be a separate PR as well - create a subissue first for a given page, make a branch off of that PR, then make a PR for the changes to that single page.
